CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.0.1] - 2019-07-23
```

Comment:
Till we dont know exact date of the release, use `2019-XX-YY`

Reply:
@paulpalamarchuk, thank you for review! [Fixed](https://github.com/terraform-google-modules/terraform-google-log-export/compare/d341db8199d26d78938f1b4049c6d59c57d36fc9..e4008a86b9e340d53954672da57fbfce036d13ac)